src/corelibs/U2Formats/src/FastqFormat.cpp

```diff
@@ -478,30 +544,30 @@ DNASequence *FastqFormat::loadSequence(IOAdapter* io, U2OpStatus& os) {
     QString sequenceName = readSequenceName(os, io, '@');
     // check for eof while trying to read another FASTQ block
     CHECK_EXT(!io->isEof(), , NULL);
-    CHECK_OP(os, NULL);
+    CHECK_OP(os, new DNASequence());
```

Comment:
Alignment stops with an error if it returns NULL, and finishes correctly with new DNASequence()